automated install/basic-install.sh

```diff
@@ -1532,13 +1532,13 @@ installPihole() {
     chown ${LIGHTTPD_USER}:${LIGHTTPD_GROUP} /var/www/html
     chmod 775 /var/www/html
     # Give pihole access to the Web server group
-    usermod -a -G ${LIGHTTPD_GROUP} pihole
+    usermod -a -G ${LIGHTTPD_USER} pihole
```

Comment:
Why is this changed?

Reply:
This was actually a misunderstanding on my part -- I'll amend and remove these changes.